src/main/java/seedu/storemando/logic/commands/ReminderCommand.java

```diff
@@ -19,23 +19,49 @@
         + "Parameters: Days/Weeks (must be an integer) [TimeUnitKeyWord] (days/weeks (day/week accepted for -1/0/1)\n"
         + "Example: \n- " + COMMAND_WORD + " 3 days\n- " + COMMAND_WORD + " 1 week";
 
-    public static final String MESSAGE_SUCCESS = "Display all expiring items";
+    public static final String MESSAGE_SUCCESS_EXPIRING_ITEM = "Display all items that are expiring in %d %s or has "
+        + "already expired.";
+    public static final String MESSAGE_SUCCESS_EXPIRED_ITEM = "Display all items that has been expired for %d %s";
```

Comment:
i think itll clearer to say for example, "at least  7 days" instead of "for 7 days".

Reply:
All right noted!